src/sentry/sentry_metrics/querying/data_v2/api.py

```diff
@@ -0,0 +1,61 @@
+from collections.abc import Sequence
```

Comment:
Copied the entire code in `data` to make the new changes completely independent of the old ones. Should enable us to keep the old endpoint up and running without problems.